website/docs/d/dnssec.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "ns1"
+page_title: "NS1: ns1_dnssec"
+sidebar_current: "docs-ns1-datasource-dnssec"
+description: |-
+  Provides DNSSEC details about a NS1 Zone.
+---
+
+# Data Source: ns1_dnssec
+
+Provides DNSSEC details about a NS1 Zone.
+
+## Example Usage
+
+```hcl
+# Get DNSSEC details about a NS1 Zone.
+resource "ns1_zone" "example" {
+  zone   = "terraform.example.io"
+  dnssec = true
+}
+
+data "ns1_dnssec" "example" {
+  zone = "${ns1_zone.example.zone}"
+}
+```
+
+## Argument Reference
+
+* `zone` - (Required) The name of the zone to get DNSSEC details for.
+
+## Attributes Reference
```

Comment:
I don't really know much about these fields and how they are used so the docs are a bit vague. If you think it's too vague, open to suggestions of course

Reply:
LGTM, especially since I think people are usually just copying and pasting this data into another provider.